calico_containers/calico_ctl/checksystem.py

```diff
@@ -177,3 +181,36 @@ def _check_docker_version(libnetwork=False):
             system_ok = False
 
     return system_ok
+
+def _check_etcd_version():
```

Comment:
I'd like to see UT of this function that mocks out `api_execute` calls with some results we know occur in the field.